pages/index.js

```diff
@@ -395,7 +395,7 @@ export default function Home({ total, today }) {
 
         <div className="relative bg-white py-24 sm:py-32 lg:py-40">
           <div className="mx-auto max-w-md px-6 text-center sm:max-w-3xl lg:max-w-7xl lg:px-8">
-            <h2 className="text-lg font-semibold text-indigo-600">
+            <h2 className="text-lg font-semibold text-indigo-600 text-3xl">
```

Comment:
There is duplicate text size, `text-lg` and `text-3xl`

Reply:
Will take care of it